spec/rails_helper.rb

```diff
@@ -59,7 +59,7 @@
 Shoulda::Matchers.configure do |config|
   config.integrate do |with|
     with.test_framework :rspec
-    with.library :active_record
+    with.library :rails
```

Comment:
Extract this into its own PR

Reply:
As this is only necessary for this PR and a test only change and this will fail w/o I think we should keep it in here.